core/slim_globals.h

```diff
@@ -354,6 +354,432 @@ extern EidosValue_String_SP gStaticEidosValue_StringT;
+"        \"is_null\": {\n"
+"            \"binaryFormat\": \"B\",\n"
+"            \"description\": \"Whether this node describes a 'null' (non-existant) chromosome.\",\n"
+"            \"index\": 1,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"slim_id\": {\n"
+"            \"binaryFormat\": \"q\",\n"
+"            \"description\": \"The 'pedigree ID' of this chromosome in SLiM.\",\n"
+"            \"index\": 0,\n"
+"            \"type\": \"integer\"\n"
+"        }\n"
+"    },\n"
+"    \"required\": [\n"
+"        \"slim_id\",\n"
+"        \"is_null\",\n"
+"        \"genome_type\"\n"
+"    ],\n"
+"    \"type\": [\n"
+"        \"object\",\n"
+"        \"null\"\n"
+"    ]\n"
+"}";
+
+const std::string SLIM_TSK_INDIVIDUAL_METADATA_SCHEMA =
+"{\n"
+"    \"$schema\": \"http://json-schema.org/schema#\",\n"
+"    \"additionalProperties\": false,\n"
+"    \"codec\": \"struct\",\n"
+"    \"description\": \"SLiM schema for individual metadata.\",\n"
+"    \"examples\": [\n"
+"        {\n"
+"            \"age\": -1,\n"
+"            \"flags\": 0,\n"
+"            \"pedigree_id\": 123,\n"
+"            \"population\": 0,\n"
+"            \"sex\": 0\n"
+"        }\n"
+"    ],\n"
+"    \"flags\": {\n"
+"        \"SLIM_INDIVIDUAL_METADATA_MIGRATED\": {\n"
+"            \"description\": \"Whether this individual was a migrant, either in the generation when the tree sequence was written out (if the individual was alive then), or in the generation of the last time they were Remembered (if not).\",\n"
+"            \"value\": 1\n"
+"        }\n"
+"    },\n"
+"    \"properties\": {\n"
+"        \"age\": {\n"
+"            \"binaryFormat\": \"i\",\n"
+"            \"description\": \"The age of this individual, either when the tree sequence was written out (if the individual was alive then), or the last time they were Remembered (if not).\",\n"
+"            \"index\": 2,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"flags\": {\n"
+"            \"binaryFormat\": \"I\",\n"
+"            \"description\": \"Other information about the individual: see 'flags'.\",\n"
+"            \"index\": 5,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"pedigree_id\": {\n"
+"            \"binaryFormat\": \"q\",\n"
+"            \"description\": \"The 'pedigree ID' of this individual in SLiM.\",\n"
+"            \"index\": 1,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"population\": {\n"
+"            \"binaryFormat\": \"i\",\n"
+"            \"description\": \"The population the individual was part of, either when the tree sequence was written out (if the individual was alive then), or the last time they were Remembered (if not).\",\n"
+"            \"index\": 3,\n"
+"            \"type\": \"integer\"\n"
+"        },\n"
+"        \"sex\": {\n"
+"            \"binaryFormat\": \"i\",\n"
+"            \"description\": \"The sex of the individual (hermaphrodite/male/female).\",\n"
```

Comment:
I think not, as above?

Reply:
OK.  Maybe if we're not going to provide an enum for integer values like this it would be nice to state what the defined values are in the description text?  Of course people can look it up; but saying "(0=hermaphrodite/1=male/2=female)" (or whatever it is) instead of "(hermaphrodite/male/female)" is only a few extra characters and is informative...?  Likewise for the other enum-that-we're-not-giving-an-enum-for.  :->

Reply:
I agree, but the problem with this is that if we write the same thing down in lots of places, then if it ever *changes* we'll likely forget to go fix it in all the places we wrote it down. But, maybe this stuff is stable enough?

Reply:
Well, that's a point.  It probably is stable, but still, OK, let's keep the info in a centralized location rather than duplicating it all over, fair enough.

Reply:
Ok, I pulled these out of the manual and put them in.

Reply:
... and, wrote that reply before seeing your comment. =) I've got them in there now, shall we leave them in?

Reply:
I'm indifferent, I think